scout/server/blueprints/institutes/templates/overview/utils.html

```diff
@@ -297,6 +324,9 @@ <h3 class="header">General Institute Settings</h3>
               {%- for sample in variant.samples -%}
                 {%- for ind in case.individuals -%}
                   {%- if sample.sample_id == ind.individual_id -%}
+                    {% if sample.genotype_call not in ['0/0', './.', '0/.', './0'] %}
+                      {% do analysis_types.append(ind.analysis_type) %} <!-- used later in Analysis type column -->
```

Comment:
Nice! This may still be a bitt fuzzy on some esp older SV calls, but I can't think of a better way.